tests/comfy_cli/command/models/test_models.py

```
@@ -307,6 +307,27 @@ def test_huggingface_url_with_folder_structure():
     )
 
 
+
+class TestFormatElapsed:
```

Pylint (4.0.5)

[convention] Missing class docstring

(C0115)